mercuryseries/inertia-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "bundles": {
+        "MercurySeries\\Bundle\\InertiaBundle\\MercurySeriesInertiaBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "gitignore": ["/%PUBLIC_DIR%/build-ssr/"],
+    "add-lines": [
```

Comment:
Instead of `add-lines`, you should ship a new `config/routes/intertia.yaml` routes file with the routes inside. This configurator is meant as a last resort - but there is a built-in standard already for adding new routing files

Reply:
Done @weaverryan  thanks!
Is there a way to automatically detect which part to uncomment: https://github.com/symfony/recipes-contrib/blob/32f43347b449fa0d8c35d013621f86d3dbc0f91a/mercuryseries/inertia-bundle/0.1/config/routes/inertia.yaml?